Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -147,14 +147,12 @@ - (instancetype)initWithApp:(FIRApp *)app
                                                      analyticsManager:_analyticsManager];
 
     // Process did crash during previous execution
-    NSString *crashedMarkerFileName = [NSString stringWithUTF8String:FIRCLSCrashedMarkerFileName];
-    NSString *crashedMarkerFileFullPath =
-        [[_fileManager rootPath] stringByAppendingPathComponent:crashedMarkerFileName];
-    _didPreviouslyCrash = [_fileManager fileExistsAtPath:crashedMarkerFileFullPath];
-
-    if (_didPreviouslyCrash) {
+    if ([_fileManager didCrashOnPreviousExecution]) {
```

Comment:
We're no longer setting `self.didPreviouslyCrash`, which means the public API `didCrashDuringPreviousExecution` is gonna be wrong